x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/use_findings_by_resource.ts

```diff
@@ -104,28 +93,34 @@ export const useFindingsByResource = ({ index, query, from, size }: UseResourceF
         data.search.search<FindingsAggRequest, FindingsAggResponse>({
           params: getFindingsByResourceAggQuery({ index, query, from, size }),
         })
-      ),
+      ).then(({ rawResponse: { aggregations } }) => {
```

Comment:
Consider using `await` instead of `then`

Reply:
+1

Reply:
will do in the [next](https://github.com/elastic/security-team/issues/3591) one (overlaps anyway) and i'd prefer merge this now